src/model/mod.rs

```diff
@@ -0,0 +1,5 @@
+mod pacbuild;
+mod repository;
+
+pub use crate::model::pacbuild::*;
+pub use crate::model::repository::{default_repository, Repository};
```

Comment:
Add a module description, it's missing.

![image](https://user-images.githubusercontent.com/49679481/188493784-ff4dc926-feaf-46e0-96f5-c9f3e7425862.png)